src/app/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
This feels a bit suspicious. Did we mean to commit a nested src/app/package.json (and package-lock.json) file?